### ISSUE TITLE
sc_notify_id could not be located in the dynamic link library

### ISSUE THREAD

**Issue by flickerfly** [OpenSC/OpenSC#1980]
### Problem Description

When I restart my windows 10 computer I have a bunch of pop up and disappear cascading down my primary display. During this I get a message that says, "The procedure entry point sc_notify_id could not be located in the dynamic link library C:\Program Files\OpenSC Project\OpenSC\tools\opensc-notify.exe". My CAC seems to work with Firefox at least so things work, but every restart I have to wait out that stuff. Also, I see the file mentioned in my directory so it is present.

An aside: I'm afraid I can't find the document mentioned in this report form for review 
"Please read about [reporting bugs](https://github.com/OpenSC/OpenSC/wiki/How-to-report-bugs-so-that-they-can-be-fixed) before opening an issue." Perhaps the link should be to https://github.com/OpenSC/OpenSC/wiki/How-to-write-a-good-bug-report?

### Proposed Resolution

None

### Steps to reproduce

Install OpenSC-0.20.0_win64 on Windows 10 Enterprise Version 10.0.18363 Build 18363.
Reboot 

### Logs

I made the changes to opensc.conf and rebooted, but I don't know where to find the file opensc-debug.log. I expected it to be in the same folder as opensc.conf. It either isn't or didn't get created. 

When trying to use opensc-tool.exe to [get the version as mentioned in the wiki](https://github.com/OpenSC/OpenSC/wiki/How-to-write-a-good-bug-report#version-number), it reports an unrecognized option

```
PS C:\Program Files\OpenSC Project\OpenSC\tools> .\opensc-tool.exe --version
C:\Program Files\OpenSC Project\OpenSC\tools\opensc-tool.exe: unrecognized option `--version'
```

However, --info seems to give me some of the output that --version was expected to provide

```
PS C:\Program Files\OpenSC Project\OpenSC\tools> .\opensc-tool.exe --info
opensc 0.13.0 [Microsoft 1600]
Enabled features:pcsc openssl zlib
```

**Comment by frankmorgner** [comment]
Do you have multiple versions of OpenSC installed? Did you try uninstalling and installing OpenSC?

**Comment by frankmorgner** [comment]
On Windows, you can run the following as administrator in a terminal to see missing libraries:
```
sfc /scannow
```

**Comment by flickerfly** [comment]
I did an uninstall and then re-installed from a fresh downloaded release. It had the same name and file size as the previous download, but this time when I issue `opensc-tool.exe --info` I get the correct version (0.20.0) instead of 0.13.0. Also, the error went away on reboot. I did also run the `sfc /scannow` for good measure afterwards and that also seems better. 

```
C:\WINDOWS\system32>sfc /scannow
Beginning system scan.  This process will take some time.

Beginning verification phase of system scan.
Verification 100% complete.

Windows Resource Protection did not find any integrity violations.
```

Thanks @frankmorgner!

I also appreciate the good product.
